extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1547,6 +1555,31 @@ export class AmpA4A extends AMP.BaseElement {
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
 
+  tryExecuteRealTimeConfig_() {
+    if (!!AMP.RealTimeConfigManager) {
+      try {
+        this.realTimeConfigManager_ = new AMP.RealTimeConfigManager(
+            this.element, this.win, this.getAmpDoc());
+        if (this.realTimeConfigManager_.validateRtcConfig()) {
+          return this.realTimeConfigManager_.executeRealTimeConfig(
+              this.getCustomRealTimeConfigMacros_());
+        };
+      } catch (err) {
+        dev().error(TAG, 'Could not perform Real Time Config');
+      }
+    }
+    // If RTC can't be performed, pass null in the resolution.
+    return Promise.resolve(null);
```

Comment:
I don't think you need this... it will just result in undefined being returned which will be passed to getAdUrl.  So long as getAdUrl signature understands this case, it should be fine (an no different than null)